Makefile

```diff
@@ -58,7 +58,7 @@ target/wasm32-wasi/$(TARGET)/img.tar: target/wasm32-wasi/$(TARGET)/wasi-demo-app
 	cd crates/wasi-demo-app && cargo build $(RELEASE_FLAG) --features oci-v1-tar
 
 load: target/wasm32-wasi/$(TARGET)/img.tar
-	sudo ctr -n $(CONTAINERD_NAMESPACE) image import --all-platforms $<
+	sudo ctr -n $(CONTAINERD_NAMESPACE) image import --all-platforms target/wasm32-wasi/debug/img.tar
```

Comment:
Why this change?
With the change it doesn't take into account `$TARGET` but `$<` does.

Reply:
It was a leftover from a rebase... fixed now.